lib/ansible/plugins/filter/basename.yml

```diff
@@ -0,0 +1,21 @@
+DOCUMENTATION:
+  name: basename
+  author: ansible core team
+  version_added: "historical"
+  short_description: get a path's base name
+  description:
+    - Returns the last name component of a path, what is left in the string that is not 'dirname'.
```

Comment:
Can `dirname` and `basename` link to each other in some fashion?

Reply:
Nothing specific, it just feels like there should be a link, since they each reference the other in the documentation text.

Reply:
Instead of inventing a new syntax, why not stick to the one planned for semantic markup (https://hackmd.io/VjN60QSoRSSeRfvGmOH1lQ?both)? :)

It would be:
```
seealso:
  - plugin: dirname
    plugin_type: filter
```

Reply:
The way to do this today, without a new syntax (which would be very welcome!), is to use references. I use this extensively in my collection to link to lookup plugins: 
https://github.com/ansible-collections/community.hashi_vault/blob/b63afc36559978fdc2745f9362114b7c9e515e12/plugins/modules/vault_kv1_get.py#L21-L22

Reply:
TIL, did not know this was added already

Reply:
It's not yet (because semantic markup in general is delayed indefinitely), but I'll add it very soon (i.e. next days).

Reply:
@bcoca the format I linked to works now, it's just a pain because you need to know the RST reference and fill in the title and description yourself. The module-like format Felix mentioned would be much nicer.

Reply:
Implementation in antsibull-docs: https://github.com/ansible-community/antsibull-docs/pull/8